.gitignore

```diff
@@ -1,8 +1,9 @@
 # folders
-build/
-build*/
+build/*
+!build/.gitkeep
```

Comment:
Does this still cover the `build*/` case? I'm not that familiar with the gitignore syntax.

Reply:
I think it doesn't, but I'm also unsure if build*/ dirs are ever exist. I thought all of the generated code is put into the build dir.

Do you have an example where build*/ is generated?

Reply:
I often have multiple `build_*` directories when I test stuff with different configurations, e.g. a build folder for ntuples and a build folder for friends which take the produced ntuples. Or for comparisons of different ntuple configurations.

Reply:
Added 'build*/' back.